GlossaryOfTerms.ttl

```diff
@@ -5337,3 +5338,21 @@ owl:IrreflexiveProperty
                             <http://data.openlinksw.com/oplweb/glossary-term/DataLake#this> ;
     schema:mainEntityOfPage <https://martinfowler.com/articles/data-monolith-to-mesh.html> ;
     skos:inScheme           :this .
+
+## Digital Twin
+
+<http://data.openlinksw.com/oplweb/glossary-term/DigitalTwin#this>
+    a skos:Concept;
+    schema:name "Digital Twin"@en ;
+    schema:description """A digital twin is a virtual representation that serves as the real-time digital 
+                          counterpart of a physical object or process. The digital twin acts as a proxy 
+                          for its thing; any application that needs data from the thing deals with the 
+                          proxy instead.
```

Comment:
applying change from comments --
```suggestion
    schema:description """A digital twin is a digital representation of a physical artifact or process.
                          It may also be a higher-fidelity rendition of another digital artifact. A
                          digital twin provides a proxy for replicating interactions across digital and 
                          analog realms.
```